yt_dlp/extractor/zdf.py

```diff
@@ -108,7 +109,7 @@ def _extract_ptmd(self, ptmd_url, video_id, api_token, referrer):
                                 'class': track.get('class'),
                                 'language': track.get('language'),
                             })
-        self._sort_formats(formats, ('hasaud', 'res', 'quality', 'language_preference'))
+        self._sort_formats(formats, ('tbr', 'hasaud', 'res', 'quality', 'language_preference'))
```

Comment:
I think `hasaud`'s priority is a mistake. Can you check removing it?

Reply:
Yep, I've removed it and still works

```bash
[debug] Command-line config: ['-vU', '-F', 'https://www.zdf.de/serien/soko-stuttgart/das-geld-anderer-leute-100.html']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.05.18 [b14d52355] (source)
[debug] Lazy loading extractors is disabled
[debug] Plugins: ['SamplePluginIE', 'SamplePluginPP']
[debug] Git HEAD: a067bff20
[debug] Python version 3.8.10 (CPython 64bit) - Linux-5.4.0-117-generic-x86_64-with-glibc2.29
[debug] Checking exe version: ffmpeg -bsfs
[debug] Checking exe version: ffprobe -bsfs
[debug] exe versions: ffmpeg 4.2.7, ffprobe 4.2.7, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.14.1, brotli-1.0.9, certifi-2021.10.08, mutagen-1.45.1, sqlite3-2.6.0, websockets-10.3
[debug] Proxy map: {}
Latest version: 2022.05.18, Current version: 2022.05.18
yt-dlp is up to date (2022.05.18)
[debug] Using fake IP 53.164.212.11 (DE) as X-Forwarded-For
[debug] [ZDF] Extracting URL: https://www.zdf.de/serien/soko-stuttgart/das-geld-anderer-leute-100.html
[ZDF] das-geld-anderer-leute-100: Downloading webpage
[ZDF] das-geld-anderer-leute-100: Downloading JSON content
[ZDF] das-geld-anderer-leute-100: Downloading JSON metadata
[ZDF] 191205_1800_sendung_sok8: Downloading m3u8 information
[ZDF] 191205_1800_sendung_sok8: Downloading m3u8 information
[ZDF] 191205_1800_sendung_sok8: Downloading m3u8 information
[ZDF] 191205_1800_sendung_sok8: Downloading m3u8 information
[debug] Sort order given by extractor: tbr, res, quality, language_preference
[debug] Formats sorted by: hasvid, ie_pref, tbr, res, quality, lang, fps, hdr:12(7), vcodec:vp9.2(10), acodec, filesize, fs_approx, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[info] Available formats for 191205_1800_sendung_sok8:
ID                                      EXT RESOLUTION FPS │   FILESIZE   TBR PROTO  │ VCODEC        VBR ACODEC     ABR MORE INFO
─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
hls-audio0-Audiodeskription-0           mp4 audio only     │                  m3u8_n │ audio only        unknown        [de] Audiodeskription
hls-audio0-TV_Ton-0                     mp4 audio only     │                  m3u8_n │ audio only        unknown        [de] TV Ton
hls-audio0-Audiodeskription-1           mp4 audio only     │                  m3u8_n │ audio only        unknown        [de] Audiodeskription
hls-audio0-TV_Ton-1                     mp4 audio only     │                  m3u8_n │ audio only        unknown        [de] TV Ton
hls-audio0-Audiodeskription-2           mp4 audio only     │                  m3u8_n │ audio only        unknown        [de] Audiodeskription
hls-audio0-TV_Ton-2                     mp4 audio only     │                  m3u8_n │ audio only        unknown        [de] TV Ton
hls-audio0-Audiodeskription-3           mp4 audio only     │                  m3u8_n │ audio only        unknown        [de] Audiodeskription
hls-audio0-TV_Ton-3                     mp4 audio only     │                  m3u8_n │ audio only        unknown        [de] TV Ton
hls-381-0                               mp4 480x272     25 │ ~120.28MiB  381k m3u8_n │ avc1.4d401f  381k video only  0k [deu] auto, ad
hls-381-1                               mp4 480x272     25 │ ~120.28MiB  381k m3u8_n │ avc1.4d401f  381k video only  0k [deu] auto, ad
hls-381-2                               mp4 480x272     25 │ ~120.28MiB  381k m3u8_n │ avc1.4d401f  381k video only  0k [deu] auto, main
hls-381-3                               mp4 480x272     25 │ ~120.28MiB  381k m3u8_n │ avc1.4d401f  381k video only  0k [deu] auto, main
hls-381-4                               mp4 480x272     25 │ ~120.28MiB  381k m3u8_n │ avc1.4d401f  381k video only  0k [deu] med, ad
hls-381-5                               mp4 480x272     25 │ ~120.28MiB  381k m3u8_n │ avc1.4d401f  381k video only  0k [deu] med, ad
hls-381-6                               mp4 480x272     25 │ ~120.28MiB  381k m3u8_n │ avc1.4d401f  381k video only  0k [deu] med, main
hls-381-7                               mp4 480x272     25 │ ~120.28MiB  381k m3u8_n │ avc1.4d401f  381k video only  0k [deu] med, main
http-h264_aac_mp4_http_na_na-low-0      mp4 unknown        │ ~149.97MiB  476k https  │ h264         476k aac         0k [deu] low, ad
http-h264_aac_mp4_http_na_na-low-1      mp4 unknown        │ ~149.97MiB  476k https  │ h264         476k aac         0k [deu] low, main
hls-614-0                               mp4 640x360     25 │ ~193.56MiB  614k m3u8_n │ avc1.4d401f  614k video only  0k [deu] auto, ad
hls-614-1                               mp4 640x360     25 │ ~193.56MiB  614k m3u8_n │ avc1.4d401f  614k video only  0k [deu] auto, main
hls-614-2                               mp4 640x360     25 │ ~193.56MiB  614k m3u8_n │ avc1.4d401f  614k video only  0k [deu] med, ad
hls-614-3                               mp4 640x360     25 │ ~193.56MiB  614k m3u8_n │ avc1.4d401f  614k video only  0k [deu] med, main
http-h264_aac_mp4_http_na_na-high-0     mp4 unknown        │ ~244.49MiB  776k https  │ h264         776k aac         0k [deu] high, ad
http-h264_aac_mp4_http_na_na-high-1     mp4 unknown        │ ~244.49MiB  776k https  │ h264         776k aac         0k [deu] high, ad
http-h264_aac_mp4_http_na_na-high-2     mp4 unknown        │ ~244.49MiB  776k https  │ h264         776k aac         0k [deu] high, main
http-h264_aac_mp4_http_na_na-high-3     mp4 unknown        │ ~244.49MiB  776k https  │ h264         776k aac         0k [deu] high, main
hls-1181-0                              mp4 852x480     25 │ ~372.21MiB 1181k m3u8_n │ avc1.4d401f 1181k video only  0k [deu] auto, ad
hls-1181-1                              mp4 852x480     25 │ ~372.21MiB 1181k m3u8_n │ avc1.4d401f 1181k video only  0k [deu] auto, main
hls-1181-2                              mp4 852x480     25 │ ~372.21MiB 1181k m3u8_n │ avc1.4d401f 1181k video only  0k [deu] med, ad
hls-1181-3                              mp4 852x480     25 │ ~372.21MiB 1181k m3u8_n │ avc1.4d401f 1181k video only  0k [deu] med, main
http-h264_aac_mp4_http_na_na-veryhigh-0 mp4 unknown        │ ~471.33MiB 1496k https  │ h264        1496k aac         0k [deu] veryhigh, ad
http-h264_aac_mp4_http_na_na-veryhigh-1 mp4 unknown        │ ~471.33MiB 1496k https  │ h264        1496k aac         0k [deu] veryhigh, ad
http-h264_aac_mp4_http_na_na-veryhigh-2 mp4 unknown        │ ~471.33MiB 1496k https  │ h264        1496k aac         0k [deu] veryhigh, main
http-h264_aac_mp4_http_na_na-veryhigh-3 mp4 unknown        │ ~471.33MiB 1496k https  │ h264        1496k aac         0k [deu] veryhigh, main
hls-1827-0                              mp4 1024x576    25 │ ~575.84MiB 1827k m3u8_n │ avc1.4d401f 1827k video only  0k [deu] auto, ad
hls-1827-1                              mp4 1024x576    25 │ ~575.84MiB 1827k m3u8_n │ avc1.4d401f 1827k video only  0k [deu] auto, main
hls-2641-0                              mp4 1280x720    50 │ ~832.26MiB 2641k m3u8_n │ avc1.640028 2641k video only  0k [deu] auto, ad
hls-2641-1                              mp4 1280x720    50 │ ~832.26MiB 2641k m3u8_n │ avc1.640028 2641k video only  0k [deu] auto, main

```